gesund/core/_metrics/classification/create_validation.py

```diff
@@ -154,6 +154,9 @@ def plot_metrics(
 
         :return: None
         """
+        os.makedirs(plot_outputs_dir, exist_ok=True)
```

Comment:
@ozkanuysal i think we can move this to Validation class under core/validation/_validation.py and things will be the same.